pkg/operator/ceph/controller/object_operations.go

```diff
@@ -32,20 +33,25 @@ func CreateOrUpdateObject(client client.Client, obj client.Object) error {
 	if err != nil {
 		return errors.Wrap(err, "failed to get meta information of object")
 	}
+	objName := accessor.GetName()
+
+	// Somehow meta.TypeAccessor returns an empty string for the type name so using reflection instead
+	objType := reflect.TypeOf(obj)
```

Comment:
If this is only in unit tests, that is to be expected, and I've had to manually fill this info in in the tests.

Reply:
As discussed, it was at runtime :(